samples/sample-app/src/App.js

```diff
@@ -19,8 +18,15 @@ function App() {
   const onSuccess = res => {
     console.log("Success");
     console.log(res);
+	setUploadedImageSource(res.url);
```

Comment:
These changes should not be in sample-app.

Reply:
This change is to show the functionality in same app and not for test cases

Reply:
Will be addressed in #42